blueprints/ember-electron/index.js

```diff
@@ -24,96 +21,143 @@ class EmberElectronBlueprint extends Blueprint {
-    let task = this.taskFor('npm-install');
-
-    logger.startProgress('Installing electron tools (electron-forge + electron-protocol-serve)');
-
-    // There are two parts of electron-forge's import command that are relevant to us.
-    // One is installing dependencies, and the other is setting up the electron-forge configuration.
-    // Rather than embedding the forge configuration inline in package.json
-    // we want it in an external file, so we install the dependencies here ourselves
-    // (including our own dependencies), and add an entry to package.json
-    // pointing the forge config to the config file that is copied from the blueprint files.
-
-    const { devDeps, exactDevDeps, deps } = require('electron-forge/dist/init/init-npm');
-
-    return task.run({
-      'save-dev': true,
-      verbose: false,
-      packages: devDeps
-    }).then(() => task.run({
-      // Dependencies we need to keep locked at specific versions
-      'save-dev': true,
-      'save-exact': true,
-      verbose: false,
-      packages: exactDevDeps
-    })).then(() => task.run({
-      // Production dependencies
-      save: true,
-      verbose: true,
-      packages: ['electron-protocol-serve@^1.3.0', ...deps]
-    }));
+  _installElectronTooling(logger) {
+    // n.b. addPackageToProject does not let us save prod deps, so we task
+    let npmInstall = this.taskFor('npm-install');
+
+    logger.startProgress('Installing electron build tools');
+
+    return efImport({
+      updateScripts: false,
+      outDir: 'electron-out',
+    })
+      .then(() => npmInstall.run({
+        saveDev: true,
+        verbose: false,
+        packages: ['devtron@^1.4.0'],
+      }))
+      .then(() => npmInstall.run({
+        save: true,
+        verbose: false,
+        packages: ['electron-protocol-serve@^1.3.0'],
+      }))
+      .then(() => logger.message('Installed electron build tools'));
   }
 
-  _createResourcesDirectories() {
-    const platforms = ['', 'darwin', 'linux', 'win32'];
-    let promises = platforms.map((platform) => {
-      let gitKeepPath = path.join(this.options.project.root, 'ember-electron', `resources${platform ? '-' : ''}${platform}`, '.gitkeep');
-      return ensureFile(gitKeepPath);
-    });
+  _createResourcesDirectories(logger) {
+    const ensureFile = denodeify(fs.ensureFile);
 
-    return all(promises);
+    logger.startProgress('Creating ember-electron resource dirs');
+
+    let rootDir = this.options.project.root;
+    let ensureResourceDirGitKeeps = [
+      'resources',
+      'resources-darwin',
+      'resources-linux',
+      'resources-win32',
+    ]
+      .map((dirName) => path.join(rootDir, 'ember-electron', dirName))
+      .map((dirPath) => ensureFile(path.join(dirPath, '.gitkeep')));
+
+    return all(ensureResourceDirGitKeeps)
+      .then(() => logger.message('Created ember-electron resource dirs'));
   }
 
-  _ensurePackageJsonConfiguration() {
+  _ensureForgeConfig(logger) {
```

Comment:
Let’s please have this discussion in an issue or a PR to the gh-pages branch. It will be much easier to make the right decision when we have a sense of the remaining docs, and I don’t believe this is a blocker functionally.

I am otherwise working on a test branch that will resolve this concern by documenting in spec.